test/go-tests/backuprestore_test.go

```diff
@@ -142,6 +126,13 @@ func Test_BackupRestore(t *testing.T) {
 	_, err = ExecuteCommandf("keptn delete project %s", keptnProjectName)
 	require.Nil(t, err)
 
+	//clientset, _ := keptnkubeutils.GetClientset(false)
+	//assert.Eventually(t, func() bool {
+	//	return wait.PollImmediate(time.Second*3, time.Minute*5, checkDeployment(clientset, serviceName, keptnProjectName+"-dev", WaitForDeploymentOptions{WithImageName: "ghcr.io/podtato-head/podtatoserver:v0.1.0"})) != nil
+	//}, time.Minute*time.Duration(10), time.Second)
+
+	//time.Sleep(10 * time.Minute)
```

Comment:
please, remove this code if not needed

Reply:
sure, this is not prepared yet for review and merging